### PR TITLE
Testing/error handling

### PR DESCRIPTION
**Pull request by CorCanavan**
## Summary:
- This PR updates testing with sad paths accounting for 400 and 500 errors for daily quote and the dropdown on the home page. 
- This PR refactors intercept URLs to stub responses, as well as adds in status codes. 

## Type of Changes:
- [ ] Bug fix
- [X] Refactor
- [X] New feature

## How was this tested:
Cypress

### CODE REVIEW

**Review by abodnar1 — COMMENTED** [review]
Nice work! Curious to hear about your changes with the intercept and what we were doing wrong.

File: cypress/e2e/home_spec.cy.ts [hunk 1]
```diff
@@ -2,27 +2,47 @@
...
 
-  it('should have daily quote on home page load', () => {
-    cy.get('.daily-quote-wrapper').contains('h2', "The most important thing in communication is to hear what isn't being said.")
-    cy.get('.daily-quote-wrapper').contains('p', "Peter Drucker")
+  it('should display daily quote on home page load', () => {
+    cy.get('.daily-quote-container').contains('h2', "The most important thing in communication is to hear what isn't being said.")
+    cy.get('.daily-quote-container').contains('p', "Peter Drucker")
+  })
+
+  it('should display an error message if daily quote is unable to load due to 400 error', () => {
```
Comment (tfoxcollis) [inline cypress/e2e/home_spec.cy.ts:21]: Great job testing error handling!

File: src/Components/Form/Form.tsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ const Form = ({setAllQuotes, setError}: FormProps) => {
         </select>
       </form> 
       :
-      <p className="error-message">Uh oh! We are not in the mood. Please try again later.</p>
+      <p className="error-message">Uh oh! We are not in the mood. Please try again later. {error}</p>
```
Comment (tfoxcollis) [inline src/Components/Form/Form.tsx:38]: Nice change here to make the error messages more consistent!

**Review by tfoxcollis — APPROVED** [review]
Thanks for tackling this, Corinne!

**Pull request merged**